sleap/gui/suggestions.py

```diff
@@ -80,8 +80,11 @@ def basic_sample_suggestion_method(
 
         for video in labels.videos:
             if sampling_method == "stride":
+                frame_increment = video.frames // per_video
+                if (frame_increment) == 0:
+                    frame_increment = per_video
                 vid_suggestions = list(
-                    range(0, video.frames, video.frames // per_video)
+                    range(0, video.frames, frame_increment)
                 )[:per_video]
```

Comment:
Add tests:
- [ ]  "stride" suggestions works on all project videos even if "stride" length is greater than video length
- [ ]  At least one sample is taken from each video if "stride" is used - regardless of video length